src/main/java/core/basesyntax/io/CustomFileWriterImpl.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.io;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+
+public class CustomFileWriterImpl implements CustomFileWriter {
+    public void write(String content, String fileName) {
+        try (BufferedWriter bw = new BufferedWriter(new java.io.FileWriter(fileName))) {

```

Comment:
fix imports

Reply:
What exactly to fix in imports?

Reply:
import FileWriter, don't write fully qualified name java.io.FileWriter